daft-parquet/benchmark.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
I ran benchmark.sh on my local machine and it only produced results for case "single". Could you please change the script so it also measures the "partitioned" case?

Reply:
Add support for running benchmarks in both single and partitioned case